package.json

```diff
@@ -53,6 +55,7 @@
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
     "@ianvs/prettier-plugin-sort-imports": "4.1.0",
+    "@mui/icons-material": "^5.14.9",
```

Comment:
```suggestion
```

Reply:
This has been done in my recent commit, thanks for suggesting it!